src/apps/dashboard/features/backups/components/RestoreConfirmationDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import globalize from 'lib/globalize';
+import React, { FunctionComponent } from 'react';
+
+type IProps = {
+    open: boolean;
+    onClose: () => void;
+    onConfirm: () => void;
+};
+
+const RestoreConfirmationDialog: FunctionComponent<IProps> = ({ open, onClose, onConfirm }: IProps) => {
+    return (
+        <Dialog
+            open={open}
+            onClose={onClose}
+            maxWidth={'xs'}
+            fullWidth
+        >
+            <DialogTitle>
+                {globalize.translate('LabelRestore')}
+            </DialogTitle>
+
+            <DialogContent>
+                <DialogContentText>
+                    {globalize.translate('MessageRestoreDisclaimer')}
+                </DialogContentText>
+            </DialogContent>
+
+            <DialogActions>
+                <Button onClick={onClose} color='error'>
```

Comment:
Maybe just use the text variant for cancel? The "error" style seems a bit extreme for closing the dialog.